PocketKit/Sources/PocketKit/Settings/Components/SettingsRowButton.swift

```diff
@@ -7,7 +7,8 @@ struct SettingsRowButton: View {
     var icon: SFIconModel?
     var leadingImageAsset: ImageAsset?
     var trailingImageAsset: ImageAsset?
-    var tintColor: Color?
+    var leadingTintColor: Color = Color(.ui.black1)
+    var trailingTintColor: Color = Color(.ui.black1)
```

Comment:
nit: would these ever be inconsistent? if not, maybe we can have one `tintColor = .ui.black` and so one place to change it

Reply:
oh got it, thanks for the clarification!